projects/core_presentation.md

```diff
@@ -0,0 +1,51 @@
+Core Project Presentations
+==========================
+
+In order to increase the continuous learning of our team, and provide fodder for our marketing, we encourage Core Project Presentations
+
+At this point, there is no Template for such a presentation, but it should include the following:
+
+_NOTE: Total presentation should be between 50 minutes and 90 minutes.  Depending on the project, more time may be spent in one section than the other, but each should be considered._
+
+## Quick History and Context _(5-10 minutes)_
+
+* What was the motivation for initiating the project?
+* What is the hypothesis we’re trying to prove?
```

Comment:
Would it also be helpful to describe what the customer has stated he is looking for?  Since a project is so often a translation of a business goal and so frequently that translation is faulty, it might be a worthwhile addition to this kind of presentation.